### PR TITLE
Fix text color for DMs

### PR DESCRIPTION
**Pull request by rickyrombo**
### Description

Fixes bug where non-links weren't correctly colored.

### BEFORE
image
image

### AFTER
image
image

### CODE REVIEW

File: packages/web/src/pages/chat-page/components/ChatMessageListItem.module.css [hunk 1]
```diff
@@ -20,7 +20,6 @@
   display: inline-block;
   max-width: 448px;
   filter: var(--drop-shadow-near);
-  color: var(--static-netural);
```
Comment (rickyrombo) [inline packages/web/src/pages/chat-page/components/ChatMessageListItem.module.css:23]: turns out this was a typo anyway

File: packages/web/src/pages/chat-page/components/ChatMessageListItem.tsx [hunk 2]
```diff
@@ -207,9 +207,12 @@ export const ChatMessageListItem = (props: ChatMessageListItemProps) => {
             />
           ) : null}
           {!hideMessage ? (
-            <div className={styles.text}>
-              <UserGeneratedText>{message.message}</UserGeneratedText>
-            </div>
+            <UserGeneratedText
+              className={styles.text}
+              color={isAuthor ? 'staticWhite' : 'neutral'}
```
Comment (rickyrombo) [inline packages/web/src/pages/chat-page/components/ChatMessageListItem.tsx:212]: lowkey liked it better when it was using CSS

**Review by dylanjeffers — APPROVED** [review]
nice! if it was css then you'd have to have logic for classnames, which imo is more work than just passing a color prop?

**Comment by rickyrombo** [comment]
> nice! if it was css then you'd have to have logic for classnames, which imo is more work than just passing a color prop?

@dylanjeffers but here the color comes as a direct result of a class on the parent. Having one check for `isAuthor` at the root level and passing that class and then styling the bubble, text, links, tail, etc all based on that is much easier than having to check `isAuthor` in every subcomponent's props for various styles. Eg. I can style the bubble and use as many `Text` components as I want without having to manually style their color each time.

I would have preferred the `Text` component not automatically set the color and default to the default stylesheet, so that it's easily overridden by CSS

**Comment by dylanjeffers** [comment]
> > nice! if it was css then you'd have to have logic for classnames, which imo is more work than just passing a color prop?
> 
> 
> 
> @dylanjeffers but here the color comes as a direct result of a class on the parent. Having one check for `isAuthor` at the root level and applying the class `.isAuthor`, and then styling the bubble, text, links, tail, etc all based on that is much easier than having to check `isAuthor` in every subcomponent's props for various styles. Eg. I can style the bubble and use as many `Text` components as I want without having to manually style their color each time.
> 
> 
> 
> I would have preferred the `Text` component not automatically set the color and default to the default stylesheet, so that it's easily overridden by CSS

Ah gotcha, I can see how that would be nice, to style inner complements with css. Id like to check with folks if that's actually what our preference should be, cause it kinda breaks encapsulation a bit. It sounds like what you need is to use the "inherit" variant on text, in that cause it'll just be whatever the upper parent tells it to be!

**Pull request merged**